awx/urls.py

```diff
@@ -5,12 +5,14 @@
 from django.urls import re_path, include
 
 from awx.main.views import handle_400, handle_403, handle_404, handle_500, handle_csp_violation, handle_login_redirect
+from ansible_base.urls import urls as base_urls
 
 
 urlpatterns = [
     re_path(r'', include('awx.ui.urls', namespace='ui')),
     re_path(r'^ui_next/.*', include('awx.ui_next.urls', namespace='ui_next')),
     re_path(r'^api/', include('awx.api.urls', namespace='api')),
+    re_path(r'^api/v1/', include(base_urls)),
```

Comment:
I want to draw some broader attention to this. My gut reaction is that it can't possibly be a good idea to add endpoints to `/api/v1/`, because that used to exist in a prior version, and it was deprecated and then removed.

Reply:
Good point. I was following the docs so used v1 but that wouldn't fit within the context of AWX.